### DISCUSSION TITLE
Loading solutions from two different exodus files

### DISCUSSION THREAD

**Discussion by mangerij** [idaholab/moose#22418] [Q&A General] [answered]
Is it possible to load in two exodus files (say A.e and B.e) and take

var1 var2 var3 from A.e 
var4 var5 var6 from B.e as an initial condition (using restart)

?

**Comment by GiudGiud** [comment, marked as answer]
Hello

not directly. But you can do it with a multiapps and transfer the initialized quantities

Guillaume

**Reply by hugary1995** [reply]
I guess alternatively you can add as many SolutionUserobjects as you want... Then use SolutionAux to initialize these variables at INITIAL.
